### PR TITLE
Issue Fixed #6886 (Locale Dropdown Overlapping on Mobile View issue fixed)

### LINKED ISSUE #6886: The locale dropdown should be just the right side of the channel's dropdown on the product's page in mobile view.
**Bagisto Version:** Master

**Steps To reproduce**--

**1.** Log in as an admin on mobile.
**2.** Click on products inside the catalog tab.
**3.** Try to change the app locale.

**Expected result:** The **locale** dropdown should be just the right side of the **channel's** **dropdown** on the product's page in the **mobile** view.

**Actual result:** The locale and channel **dropdowns** are **overriding** in **mobile** view. Please find the screenshot [Here](https://prnt.sc/Akjj-rukiwiV)

### PR DESCRIPTION
**Pull request by JunaidJaved248**
## Issue Reference

#6886 
## Description

Issue Fixed => Locale Dropdown Overlapping on Mobile View 
## How To Test This?

Steps To Test--
1. Log in as an admin on mobile.
2. Click on products inside the catalog tab.
3. Try to change the app locale.

Check Here (Issue Fixed)
image

## Documentation
- [ ] My pull request requires an update on the documentation repository.

### CODE REVIEW

File: packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss [hunk 1]
```diff
@@ -1022,4 +1022,18 @@ body {
...
         }
     }
+}
+
+@media only screen and (min-width: 280px) and (max-width: 420px) {
+    .grid-container {
+        .grid-top {
+            .datagrid-filters {
+                .filter-left {
+                    div {
```
Comment (lex111) [inline packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss:1032]: In fact, there is an extra `div` here, which if you remove it, you get the same result without any new CSS: 

image

Comment (lex111) [reply packages/Webkul/Admin/src/Resources/assets/sass/partials/_default.scss:1032]: However we can move `.filter-left` to the component itself as a new prop eg.

**Comment by JunaidJaved248** [comment]
@lex111 Check now latest commit, Seems Good?

**Comment by lex111** [comment]
I'm not the maintainer of this repo, but I think it's definitely better.

**Comment by JunaidJaved248** [comment]
@lex111 Yes I know, I just ask for you checking for more suggestion. 
Anyway, Thanks for guiding for optimizing my solution of fixing this Issue.

**Comment by JunaidJaved248** [comment]
@lex111 Yes I know, I just ask for you reviewing that its seem like that as you suggested.
Anyway, Thanks for guiding for optimizing my solution of fixing this Issue.

**Comment by JunaidJaved248** [comment]
@Sanjaybhattwebkul Please check this PR.

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
Resolve the **conflicts**.

**Comment by Sanjaybhattwebkul** [comment]
@JunaidJaved248 
This is **working fine,** please find the screenshot [Here](https://prnt.sc/UpaZCDjBoTOd)
**Thanks** for your **contribution**.

**Review by Sanjaybhattwebkul — APPROVED** [review]

**Review by jitendra-webkul — APPROVED** [review]

**Pull request merged**